files/fr/web/html/link_types/preload/index.html

```diff
@@ -0,0 +1,222 @@
+
+  &lt;link rel="preload" href="fonts/cicle_fina-webfont.woff2" as="font" type="font/woff2" crossorigin&gt;
+  &lt;link rel="preload" href="fonts/zantroke-webfont.woff2" as="font" type="font/woff2" crossorigin&gt;
+
+  &lt;link href="style.css" rel="stylesheet"&gt;
+&lt;/head&gt;
+&lt;body&gt;
+  …
+&lt;/body&gt;</pre>
+
+<p>En plus de fournir un type MIME avec l'attribut <code>type</code>, ce code utilise un attribut <code>crossorigin</code> afin de s'assurer que le préchargement du <i lang="en">CORS</i> correspond à la requête de chargement de la police.</p>
+
+<h2 id="including_media">Inclure des médias</h2>
+
+<p>Une autre belle fonctionnalité de l'élément <code>&lt;link&gt;</code> concerne leur capacité à accepter les attributs {{htmlattrxref("media", "link")}}. Il peut accepter les requêtes de <a href="/fr/docs/Web/CSS/@media#media_types">types de médias</a> ou encore des <a href="/fr/docs/Web/CSS/Media_Queries/Using_media_queries"><i lang="en">media queries</i></a> complètes, ce qui vous permet de faire du préchargement <i lang="en">responsive</i> !</p>
+
+<p>Voici un exemple. Vous pouvez consulter son <a href="https://github.com/mdn/html-examples/tree/master/link-rel-preload/media">code source sur GitHub</a> ou étudier un <a href="https://mdn.github.io/html-examples/link-rel-preload/media/">exemple de démonstration</a> :</p>
+
+<pre class="brush: html">&lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+  &lt;title&gt;Exemple de préchargement responsive&lt;/title&gt;
+
+  &lt;link rel="preload" href="bg-image-narrow.png" as="image" media="(max-width: 600px)"&gt;
+  &lt;link rel="preload" href="bg-image-wide.png" as="image" media="(min-width: 601px)"&gt;
+
+  &lt;link rel="stylesheet" href="main.css"&gt;
+&lt;/head&gt;
+&lt;body&gt;
+  &lt;header&gt;
+    &lt;h1&gt;Mon site&lt;/h1&gt;
+  &lt;/header&gt;
+
+  &lt;script&gt;
+    var mediaQueryList = window.matchMedia("(max-width: 600px)");
+    var header = document.querySelector('header');
+
+    if (mediaQueryList.matches) {
+      header.style.backgroundImage = 'url(bg-image-narrow.png)';
+    } else {
+      header.style.backgroundImage = 'url(bg-image-wide.png)';
+    }
+  &lt;/script&gt;
+&lt;/body&gt;</pre>
+
+<p>Dans cet exemple nous incluons les attributs <code>media</code> dans notre élément <code>&lt;link&gt;</code> pour qu'une image plus fine soit préchargée si la personne visitant le site dispose d'un écran plus petit, et pour qu'une image plus large soit chargée sur les écrans plus larges. Pour cela, nous utilisons {{domxref("Window.matchMedia")}} et {{domxref("MediaQueryList")}} (consultez la page <a href="/fr/docs/Web/CSS/Media_Queries/Testing_media_queries">Tester les requêtes média en JavaScript</a> pour en savoir plus).</p>
+
+<p>Cela augmente les chances que la police sera disponible lors du rendu de la page, et diminue les risques de <i lang="en">FOUT</i> (pour <i lang="en">flash of unstyled text</i>, soit « flash de texte stylé » en français).</p>
```

Comment:
plutôt "flash de texte sans style" ou "flash de text sans mise en forme"
"FDTSMEF" :)

Reply:
Fixed in https://github.com/mdn/translated-content/pull/1808/commits/eac2bca7098ce774dfb3371804dc2bd109b5459b